starlark/sys_unix.go

```diff
@@ -0,0 +1,12 @@
+// +build linux darwin
+
+package starlark
+
+import "syscall"
+
+func nanotime() int64 {
+	// gettimeofday (17ns) is almost 3x faster than time.Now (46ns).
+	var tv syscall.Timeval
+	syscall.Gettimeofday(&tv) // can't fail
```

Comment:
This doesn't seem like the right clock because it could be changed while the process is running. I'm not sure what the canonical correct answer is. I would expect something based on `CLOCK_MONOTONIC`, but I'm not sure of the precision or latency there.